client/tokenized-express-checkout/compatibility/wc-deposits.js

```diff
@@ -1,15 +1,33 @@
 /* global jQuery */
+/**
+ * External dependencies
+ */
+import { addFilter, doAction } from '@wordpress/hooks';
+
 jQuery( ( $ ) => {
-	// WooCommerce Deposits support.
-	// Trigger the "woocommerce_variation_has_changed" event when the deposit option is changed.
 	$( 'input[name=wc_deposit_option],input[name=wc_deposit_payment_plan]' ).on(
 		'change',
 		() => {
-			$( 'form' )
-				.has(
-					'input[name=wc_deposit_option],input[name=wc_deposit_payment_plan]'
-				)
-				.trigger( 'woocommerce_variation_has_changed' );
+			doAction( 'wcpay.express-checkout.update-button-data' );
 		}
 	);
 } );
+addFilter(
```

Comment:
This won't really work until https://github.com/woocommerce/woocommerce-deposits/issues/665 is merged. But adding it nonetheless.